package.json

```diff
@@ -123,6 +122,19 @@
       "PR: Performance :running_woman:": ":running_woman: Performance",
       "PR: Revert :leftwards_arrow_with_hook:": ":leftwards_arrow_with_hook: Revert",
       "PR: Output optimization :microscope:": ":microscope: Output optimization"
+    },
+    "labels_breaking": {
+      "PR: Spec Compliance (Next major only) :eyeglasses:": ":eyeglasses: Spec Compliance",
```

Comment:
They can re-use the current `Babel 8` purple colorr, so it is less likely that maintainers are using incorrect labels.